components/dashboard/src/teams/JoinTeam.tsx

```diff
@@ -23,20 +23,7 @@ export default function () {
                     throw new Error("This invite URL is incorrect.");
                 }
 
-                let team;
-                try {
-                    team = await getGitpodService().server.joinTeam(inviteId);
-                } catch (error) {
-                    const message: string | undefined = error && typeof error.message === "string" && error.message;
```

Comment:
As a bonus, this removes the brittle error case handling which relies on the `slug` name being present in the unstructured error message